packages/compass-logging/src/provider.ts

```diff
@@ -110,6 +110,8 @@ export function withLoggerAndTelemetry<
   return WithLoggerAndTelemetry;
 }
 
-export function mongoLogId(id: number): MongoLogId {
+// Because dupedLogId needs to stay on the same line
+// prettier-ignore
+export function mongoLogId(id: number): MongoLogId { // !dupedLogId
```

Comment:
Not technically duped, but this is what we use to skip check-logids check in mongosh and eslint rule and I don't want to add another way of skipping it